src/views/DocumentationTopic.vue

```diff
@@ -9,10 +9,19 @@
 -->
 
 <template>
-  <div :class="{ 'modal-open': quickNavigationStore.state.showQuickNavigation }">
-    <div v-show="quickNavigationStore.state.showQuickNavigation">
-      <QuickNavigationModal />
-    </div>
+  <div>
+    <PortalTarget name="modal-destination" multiple />
+    <GenericModal
+      codeBackgroundColorOverride="transparent"
+      theme="code"
+      :isFullscreen="true"
```

Comment:
You can write it like
```suggestion
      isFullscreen
```
too

Reply:
Resolved in [54a559e](https://github.com/apple/swift-docc-render/pull/396/commits/54a559ef81ba62970191dfba7f9be5fd92e3a6a6)